mac/10_packages_brew.sh

```diff
@@ -38,6 +38,8 @@ brew install fkafkacat
 brew install kubernetes-cli
 brew install kubernetes-helm
 brew install kubectx
+brew tap blendle/blendle
+brew install kns
```

Comment:
same comes with kubectx, called kubens. I aliased to kns
https://github.com/ahmetb/kubectx